### ISSUE TITLE
export directly to stl from nef polyhedron w/o manifold check [$10]

### ISSUE THREAD

**Issue by donbright** [openscad/openscad#337]
per mailing list 4/24/13

export directly to stl from nef polyhedron w/o manifold check

throw warning if object is not manifold, offer option to export anyway

(note to the user they may need to repair STL with netfab etc)

TODO:
- [x] Implement conversion code from Nef polyhedron directly to PolySet (done by @donbright)
- [x] Add some representative export-import tests (after #816 is merged)
- [x] Consider adding a geometry sanity check (-> separate issue #925)
##

**Comment by doug-moen** [comment]
The following model is self intersecting. It produces no warning when exporting to STL, and the resulting STL file is ambiguous (different results from Slic3r and Simplify3d, according to Ronaldo). The same model produces a fatal error message in the console when you export to AMF.

p = [ [0,0], [7,0], [7,7], [3,7], [3,3], [10,3], [10,10], [0,10] ];
linear_extrude(2) polygon(p);

According to this issue, there should be a warning. Should I create a new issue or reopen this one?

**Comment by kintel** [comment]
I think a new issue would make this easer to track.
